src/documenti/esterni/AdR/contenuti/modifiche.tex

```diff
@@ -8,10 +8,12 @@
   \hline
   \rowcolor[HTML]{036400}
   \textcolor[HTML]{FFFFFF}{\textbf{Versione}} & \textcolor[HTML]{FFFFFF}{\textbf{Data}} & \textcolor[HTML]{FFFFFF}{\textbf{Autore}}  & \textcolor[HTML]{FFFFFF}{\textbf{Verificatore}} & \textcolor[HTML]{FFFFFF}{\textbf{Modifica}}    \\ \hline
+  \rowcolor[HTML]{EFEFEF}
+  v1.0.0           & ??/??/2022   & Marco Mazzucato & - & Approvazione     \\ \hline
   \rowcolor[HTML]{C0C0C0}
-  v1.0.0           & 24/02/2022   & Marco Mazzucato & - & Approvazione     \\ \hline
+  v0.1.1           & 04/03/2022    & Mattia Zanellato &                       & Modifiche sezione "UC6", relative sottosezioni e corrispondenti requisiti \\ \hline
```

Comment:
Avevamo deciso di ripartire da 0.1.0 o da 0.0.14 perchè Emanuele ha aggiunto 0.0.15...

Reply:
In effetti questo caso non lo avevamo considerato però va bene continuare da 0.0.15